src/include/daos_srv/bio.h

```diff
@@ -94,7 +94,10 @@ struct bio_dev_state {
 	uint64_t	 bds_timestamp;
 	uint64_t	*bds_media_errors; /* supports 128-bit values */
 	uint64_t	 bds_error_count; /* error log page */
-	uint32_t	 bds_bio_err;
+	/* I/O error counters */
+	uint32_t	 bds_bio_read_errs;
```

Comment:
Stlye: bds_bio sounds repetitive as b in bds is bio, how about just bds_read_errs etc?